core/inference_utils/vocabulary.py

```diff
@@ -46,8 +46,10 @@ def __init__(self,
         with tf.gfile.GFile(vocab_file, mode="r") as f:
             reverse_vocab = list(f.readlines())
         reverse_vocab = [line.split()[0] for line in reverse_vocab]
-        assert start_word in reverse_vocab
-        assert end_word in reverse_vocab
+        if start_word not in reverse_vocab:
+            ValueError("Start word '%s' is not in 'reserved_vocab" % start_word)
```

Comment:
```suggestion
            raise ValueError("Start word '%s' is not in 'reserved_vocab" % start_word)
```

Reply:
Same for all in `core/inference_utils/caption_generator.py`

Reply:
Thanks!